### DISCUSSION TITLE
IBAX cüzdanında bulunan 10000 IBX hakkında

### DISCUSSION THREAD

**Discussion by Emredurmus27** [IBAX-io/go-ibax#118] [Ideas]
Arkadaşlar cüzdanda 10.000 IBX var ama puanım 24.000 bunun hakkında bilgisi olan var mı acaba

**Comment by Silopili** [comment]
Test puanları onlar

**Reply by Emredurmus27** [reply]
Nasıl yani hocam bir getirisi olmayacak mı herkeste mı 10.000 var

**Reply by Furkanae** [reply]
Bu test puanlarını herhangi bir yerde kullanacak mıyız? Hacim falan yapmamız gerekiyor mu ?

**Comment by byblckd** [comment]
> Arkadaşlar cüzdanda 10.000 IBX var ama puanım 24.000 bunun hakkında bilgisi olan var mı acaba

cüzdan tesnet aşamasında cüzdan da görünen 10 000 coin sizlerin cüzdanı test etmesi amacıyla gönderilen tesnet coinleridir.Herhangi bir maddi değeri yoktur

**Comment by cartvecurt** [comment]
Sitedeki fonksiyonları test etmen için bir bakiye, senin olmuyor onlar.
